### ISSUE TITLE
Amazon Linux AMI - Possible False Positive for Missing Security Packages

### ISSUE THREAD

**Issue by virtualjj** [CISOfy/lynis#339]
Despite performing updates for security:

![image](https://cloud.githubusercontent.com/assets/10323889/21223256/a55e4b24-c309-11e6-9639-970addeb73b1.png)

Lynis reports a warning for missing security packages:

![image](https://cloud.githubusercontent.com/assets/10323889/21223332/fa40ce32-c309-11e6-8730-756e78ae425f.png)

uname output:

`Linux ip-172-30-6-130 4.4.35-33.55.amzn1.x86_64 #1 SMP Tue Dec 6 20:30:04 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`

AWS Instance AMI ID: 

`ami-d41d58a7`

**Comment by mboelen** [comment]
And what do you get to see in your lynis.log file?

**Comment by virtualjj** [comment]
Apologies for the delay.

I ran _**yum update --security -y**_, rebooted, and ran Lynis again. I found the same warning - here is what I see in the lynis.log file:

2016-12-18 12:48:56 Performing test ID PKGS-7386 (Check for YUM security package)
2016-12-18 12:48:56 Test: Determining if yum-security package installed
2016-12-18 12:48:56 Test: checking if file /usr/share/yum-cli/cli.py exists
2016-12-18 12:48:56 Result: file /usr/share/yum-cli/cli.py exists
2016-12-18 12:48:56 Test: search string \-\-security in file /usr/share/yum-cli/cli.py
2016-12-18 12:48:56 Result: found search string '\-\-security'
2016-12-18 12:48:56 Full string returned:         group.add_option("--security", action="store_true", 
2016-12-18 12:48:56 Result: found built-in security in yum
2016-12-18 12:48:56 Test: Checking for vulnerable packages
2016-12-18 12:48:57 Result: found vulnerable package(s)
**2016-12-18 12:48:57 Vulnerable package: lcms2-2.8-2.el6.x86_64**,
2016-12-18 12:48:57 Hardening: assigned partial number of hardening points (1 of 2). Currently having 101 points (out of 137)
**2016-12-18 12:48:57 Vulnerable package: lcms2-devel-2.8-2.el6.x86_64**,
2016-12-18 12:48:57 Hardening: assigned partial number of hardening points (1 of 2). Currently having 102 points (out of 139)
2016-12-18 12:48:57 Warning: Found one or more vulnerable packages. [test:PKGS-7386] [details:-] [solution:-]
2016-12-18 12:48:57 Suggestion: Use 'yum --security update' to update your system [test:PKGS-7386] [details:-] [solution:-]

I'm probably missing something easy but I'm not seeing an update for this:

![image](https://cloud.githubusercontent.com/assets/10323889/21293642/0fe40f72-c56e-11e6-9500-0332a3d7efcd.png)

It looks like the package is for [Little CMS](http://www.littlecms.com/download.html) and from what I can tell the version installed is the current version.
